### ISSUE TITLE
'Cannot find module' error when using version 2.0.0 with the example app

### ISSUE THREAD

**Issue by michael-wolfenden** [nikku/karma-browserify#86]
I'm trying to use this library for my own project and I'm getting a `Cannot find module` error so I thought I would try the example project.

The example worked but I noticed the package.json references an old version of karma-browser `"karma-browserify": "^1.0.0"`. When I bump this up to the latest version (`"karma-browserify": "^2.0.0"`), the example fails with the following output (the same as I am seeing with my own project).

NOTE: I'm using windows

```
DEBUG [plugin]: Loading karma-* from C:\development\karma-browserify\example\node_modules
DEBUG [plugin]: Loading plugin C:\development\karma-browserify\example\node_modules/karma-browserify.
DEBUG [plugin]: Loading plugin C:\development\karma-browserify\example\node_modules/karma-jasmine.
DEBUG [plugin]: Loading plugin C:\development\karma-browserify\example\node_modules/karma-phantomjs-launcher.
DEBUG [framework.browserify]: created browserify bundle: C:\Users\MICHAE~1\AppData\Local\Temp\7dddbaf462d41eebc1710afe4712e1631bba466a.browserify
DEBUG [framework.browserify]: add bundle to config.files at position 3
INFO [karma]: Karma v0.12.31 server started at http://localhost:9876/
INFO [launcher]: Starting browser PhantomJS
DEBUG [temp-dir]: Creating temp dir at C:\Users\MICHAE~1\AppData\Local\Temp\karma-98804596
DEBUG [launcher]: C:\development\karma-browserify\example\node_modules\karma-phantomjs-launcher\node_modules\phantomjs\lib\phantom\phantomjs.exe C:\Users\MICHAE~1\AppData\Local\Temp\karma-98804596/capture.js
DEBUG [framework.browserify]: building bundle
DEBUG [framework.browserify]: updating test\bSpec.js in bundle
DEBUG [framework.browserify]: updating test\externalSpec.js in bundle
DEBUG [framework.browserify]: bundling
INFO [framework.browserify]: 3921 bytes written (0.39 seconds)
INFO [framework.browserify]: bundle built
DEBUG [watcher]: Resolved files:
  C:/development/karma-browserify/example/node_modules/karma-jasmine/lib/jasmine.js
  C:/development/karma-browserify/example/node_modules/karma-jasmine/lib/adapter.js
  C:/development/karma-browserify/example/vendor/external.js
  C:/Users/MICHAE~1/AppData/Local/Temp/7dddbaf462d41eebc1710afe4712e1631bba466a.browserify
  C:/development/karma-browserify/example/test/bSpec.js
  C:/development/karma-browserify/example/test/externalSpec.js
DEBUG [web-server]: serving: C:\development\karma-browserify\example\node_modules\karma\static/client.html
DEBUG [web-server]: serving: C:\development\karma-browserify\example\node_modules\karma\static/karma.js
DEBUG [web-server]: upgrade /socket.io/1/websocket/ilUUNsoWl4PvKVOiIH1k
DEBUG [karma]: A browser has connected on socket ilUUNsoWl4PvKVOiIH1k
INFO [PhantomJS 1.9.8 (Windows 8)]: Connected on socket ilUUNsoWl4PvKVOiIH1k with id 98804596
DEBUG [launcher]: PhantomJS (id 98804596) captured in 3.45 secs
DEBUG [web-server]: serving: C:\development\karma-browserify\example\node_modules\karma\static/context.html
DEBUG [web-server]: serving (cached): C:/development/karma-browserify/example/node_modules/karma-jasmine/lib/jasmine.js
DEBUG [web-server]: serving (cached): C:/development/karma-browserify/example/node_modules/karma-jasmine/lib/adapter.js
DEBUG [web-server]: serving (cached): C:/development/karma-browserify/example/vendor/external.js
DEBUG [web-server]: serving (cached): C:/Users/MICHAE~1/AppData/Local/Temp/7dddbaf462d41eebc1710afe4712e1631bba466a.browserify
DEBUG [web-server]: serving (cached): C:/development/karma-browserify/example/test/bSpec.js
DEBUG [web-server]: serving (cached): C:/development/karma-browserify/example/test/externalSpec.js
[31mPhantomJS 1.9.8 (Windows 8) ERROR
  Error: Cannot find module 'C:/development/karma-browserify/example/test/bSpec.js'
  at C:/Users/MICHAE~1/AppData/Local/Temp/7dddbaf462d41eebc1710afe4712e1631bba466a.browserify:1:0 <- node_modules\karma-browserify\node_modules\browserify\node_modules\browser-pack\_prelude.js:1:0
PhantomJS 1.9.8 (Windows 8): Executed 0 of 0[31m ERROR (0.052 secs / 0 secs)
DEBUG [karma]: Run complete, exiting.
DEBUG [launcher]: Disconnecting all browsers
DEBUG [framework.browserify]: cleaning up
DEBUG [launcher]: Process PhantomJS exited with code 0
DEBUG [temp-dir]: Cleaning temp dir C:\Users\MICHAE~1\AppData\Local\Temp\karma-98804596
```

**Comment by bendrucker** [comment]
See #78. This is a Windows-only issue that's on master and pending release.

**Comment by nikku** [comment]
Just released the fixes with karma-browserify@3.0.0.
